app/models/follow.rb

```diff
@@ -0,0 +1,31 @@
+class Follow < ActiveRecord::Base
+  belongs_to :user
```

Comment:
Don't worry about adding this, seems like there is no actual usage not even reason to. On the mockups there is nowhere to use it (only individual Followable counts) so when that time comes we'll add it if @decabeza thinks it makes sense to put a global following count 👍 .